scripts/prepare_sector_network.py

```diff
@@ -2694,9 +2696,12 @@ def add_biomass(n, costs):
                 + bus_transport_costs * average_distance,
             )
             # Set last snapshot of e_max_pu for unsustainable solid biomass to 1 to make operational limit work
-            unsus_stores_idx = n.stores.loc[
-                n.stores.carrier == "unsustainable solid biomass"
-            ].index
+            unsus_stores_idx = n.stores.query(
+                "carrier == 'unsustainable solid biomass'"
+            ).index
+            unsus_stores_idx = unsus_stores_idx.intersection(
```

Comment:
In what cases n.stores_t.e_max_pu would not contain any columns related to the unsustainable solid biomass store?